dead_pool_reaper.go

```diff
@@ -293,6 +313,30 @@ func (r *deadPoolReaper) getUnknownPools() (map[string][]string, error) {
 	return pools, nil
 }
 
+// removeDanglingLocks adjusts the lock keys according to the lock_info numbers.
+// TODO: it's better to find where the inconsistency comes from.
+func (r *deadPoolReaper) removeDanglingLocks() error {
+	scriptArgs := make([]interface{}, 0, len(r.curJobTypes)+1) // +1 for keys count
+	scriptArgs = append(scriptArgs, len(r.curJobTypes)*2)      // lock and lock_info keys
```

Comment:
The initial capacity should be `l * 2 + 1`.

```suggestion
	keysCount := len(r.curJobTypes) * 2
	scriptArgs := make([]interface{}, 0, 1 + keysCount) // +1 for keys count
	scriptArgs = append(scriptArgs, keysCount)          // lock and lock_info keyskeys
```

Reply:
Ok, my bad. Fixed it